doc/reference/alerttypes.rst

```diff
@@ -356,7 +352,8 @@ Reports state changes in fan units
      - A fan unit has entered a non-OK state
    * - ``fanOK``
      - A fan unit has returned to an OK state
-
+   * - ``fanNew``
+     - A fan has been discovered on a Netbox
```

Comment:
I strongly disagree that the new alert types belong under the `*State` family of events. Users will typically be subscribed to these to know when these types of devices stop working (i.e. their up-state changes). This could result in getting streams of notifications every time a new Netbox is added, when one is only interested in knowing when stuff stop sowrking.

They still belong under the (potentially poorly named) `deviceState` event type, which is what a user would subscribe to if they were at all interested in receiving notifications about lifecycle events.

Reply:
Moved back under deviceState. subid wont be accessible then, but that might not be a problem